t/lib/server.lua

```diff
@@ -101,8 +101,15 @@ function _M.uri()
     -- ngx.sleep(1)
     ngx.say("uri: ", ngx.var.uri)
     local headers = ngx.req.get_headers()
-    for k, v in pairs(headers) do
-        ngx.say(k, ": ", v)
+
+    local keys = {}
+    for k, _ in pairs(headers) do
```

Comment:
Just ignore the second dumb variable if you don't need it.

```lua
for k in pairs(headers) do
      ......
end
```

Reply:
nice.

Reply:
fixed.